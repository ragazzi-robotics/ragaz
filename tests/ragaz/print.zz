def main():

    print(9.999)
    print(1979)

    var x = 2030
    print(str(x))

    var y = "Ragazzi"
    print(y)
