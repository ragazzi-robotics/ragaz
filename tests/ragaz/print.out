9.999000
1979
2030
Ragazzi
